src/Utils.php

```diff
@@ -18,6 +18,27 @@ public static function functionAvailable($func)
         return function_exists($func) && !in_array($func, $disabled);
     }
 
+    /**
+     * Check a string ends in a substring.
+     *
+     * @param string $string The parent string
+     * @param string $sub The substring
+     *
+     * @return bool
+     */
+    public static function stringEndsIn($string, $sub)
```

Comment:
This function seems to be unnecessarily complex. I think this would be simpler:

```php
public static function stringEndsWith($haystack, $needle)
{
    return substr($haystack, -strlen($needle)) === $needle;
}
```

Also, the naming of parameters as haystack and needle would be consistent with PHP's standard API.

Reply:
If you want to support empty string as needle, you'll need to check if it's length is 0:

```php
public static function stringEndsWith($haystack, $needle)
{
    $length = strlen($needle);
    if ($length === 0) {
        return true;
    }

    return substr($haystack, -$length) === $needle;
}
```

Reply:
That does make more sense, thanks